docs/cache/manage/limits.md

```diff
@@ -27,61 +27,63 @@ The limits on this page are soft limits that can be altered unless specifically
 
 ## Operations
 
-Service limits are based on the number of operations performed per second. Some cache data plane APIs can perform multiple operations in a single request. Since batch operations are more efficient, the limit cost of these APIs is discounted at a 2:1 ratio. This means every two elements will count as one operation towards the limiter. For example, a `SetAddElements` request adding one or two elements costs one operation, but with three or four elements it costs two operations, etc.
+Service limits are based on the number of operations performed per second. Some cache data plane APIs can perform multiple operations in a single request.
+
+Since batch operations are more efficient, the limit cost of these APIs is discounted at a **2:1 ratio**. This means every two elements will count as one operation towards the limiter. For example, a `SetAddElements` request adding one or two elements costs one operation, but with three or four elements it costs two operations, etc.
```

Comment:
NIT: `batch operations` => `multi-element operations`

Reply:
done in [916a011](https://github.com/momentohq/public-dev-docs/pull/602/commits/916a011d858e56cabf2cc76b75f79865707123ef)